app/models/manageiq/providers/hawkular/middleware_manager/event_catcher_mixin.rb

```diff
@@ -0,0 +1,67 @@
+# We use this to provide easy access to both the runner and specs
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
+  # extend ActiveSupport::Concern
+
+  # TODO: perhaps we need to block certain events.  Another way to do this would be to add a tag on events we want to
+  # suck into MiQ, and just fetch with the tag as criteria
+  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze # some Hash of included events
+  EXCLUDE_EVENTS = {}.freeze # TODO: some Hash of exclusions for the included events
```

Comment:
You define which events should be blacklisted like [here](https://github.com/ManageIQ/manageiq/blob/bc548376ead317bfd7ce8d9cbcb2b97bd88ce75c/gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/network_manager.rb#L67-L73)
These get seeded at install time.
Then you can filter events like [this](https://github.com/ManageIQ/manageiq/blob/bc548376ead317bfd7ce8d9cbcb2b97bd88ce75c/gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/runner.rb#L31)
They are setup [here](https://github.com/ManageIQ/manageiq/blob/3d42ae360d38f048dbc2e6e79e67cb9d01666ab7/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L19)

This way the user can manage which events should be blacklisted.

In case you want some events not to get into the system, you should do it as early as possible, i.e. right in the REST call if it support filtering, or right after that.

@lfu is that correct?